src/lib/viewers/doc/__tests__/DocBaseViewer-test.js

```diff
@@ -2380,6 +2381,18 @@ describe('src/lib/viewers/doc/DocBaseViewer', () => {
                 }),
             );
 
+            it('should not showHighlightText if file has no download permission', () => {
+                stubs.checkPermission.withArgs(docBase.options.file, PERMISSION_DOWNLOAD).returns(false);
```

Comment:
Any way we can avoid using `withArgs` here? Will make converting to Jest (someday) easier.

Reply:
I looked into this but it turns out `stubs.checkPermission` is already being used in the other tests